utils/update_arch_tiles.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python
+
+import argparse
+import sys
+import copy
+
+from lxml import etree as ET
+
+
+def add_tile_tags(arch):
+    """
+    This script is intended to modify the architecture description file to be compliant with
```

Comment:
Reword this sentence.  If someone is reading this docstring, "new" and "old" have very little meaning.

Reply:
Done.